definitions/ext-istio_service/definition.yml

```diff
@@ -22,6 +22,88 @@ synthesis:
     conditions:
     - attribute: metricName
       prefix: istio_tcp
+      
+  # Support for prometheus instrumentation provider
+  - identifier: "service_istio_io_canonical_name"
+    encodeIdentifierInGUID: true
+    name: "service_istio_io_canonical_name"
+    conditions:
+    - attribute: metricName
+      prefix: istio_request
```

Comment:
should we also filter by prometheus instrumentation.provider here? to make sure it only applies to the data we are expecting it to apply

Reply:
I looked at other entities with prometheus and I could not find any example where this was done. Hence, I left it out.